docs/cloud/getting-started.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 1
+title: Dragonfly Quickstart
+---
+
+This guide outlines the steps to create a data store instance on DragonflyDB Cloud.
+
+## Create a Data Store 
+
+1. From the **Data Stores** menu, at the top right corner of the dashboard,  click **Data Store**
+
+![dashboard](../../static/img/dashboard.png)
+
+
+2. Fill out the "Create Data Store" form:
+
+![create-datastore](../../static/img/create-datastore.png)
+
+   - **Name:** Provide a unique name for your data store.
+   - **Cloud Provider:**
+     - **AWS:** Select this option to create your data store on Amazon Web Services.
+     - **GCP:** Select this option to create your data store on Google Cloud Platform.
+   - **Region:**
+     - For AWS, you can choose from `us-east-1` and `eu-west-1`. More regions are coming soon.
```

Comment:
those regions happens to be the one staging, prod has many more, I don't think we should specify them here